cloud-functions/failed_fhir_conversion/main.py

```diff
@@ -0,0 +1,57 @@
+import json
```

Comment:
It will only be called when the FHIR converter fails once we have orchestration set up via a Google Workflow. The thinking behind making this a separate cloud function is that we want user to be able to mix and match these pieces as much as possible. Conceivably, a user might want to use the FHIR converter as a standalone thing that doesn't write failures to a bucket but just returns them as an API response, in which case this function won't be needed.